Project/Assets/ML-Agents/Examples/Crawler/Demos/ExpCrawlerDynVS.demo.meta

```diff
@@ -0,0 +1,10 @@
+fileFormatVersion: 2
```

Comment:
The names of these files is not consistent. I think it is fine to have longer names.

Reply:
no change. it truncates the demo name which is why it uses a different format. i don't like it either.

Reply:
Very problematic indeed, I expect users will eventually want to have longer names to differentiate their demo. We should allow for longer names. Can't we at least manually rename these demo files? The truncation only happens when the demo file is generated no?

Reply:
I have no idea if changing the name after it's been created causes any issues. lets talk about it next meeting.